### ISSUE TITLE
scdd reports version 0.94g

### ISSUE THREAD

**Issue by saraedum** [cddlib/cddlib#14]
@komeifukuda wrote:
```
One thing I noticed is that the version number shown with scdd is 0.94g.
This can be corrected in cddtypes.h .  Currently it is set to 0.94g.

cddtypes.h:#define dd_DDVERSION   "Version 0.94g (March 23, 2012)"
```

**Comment by saraedum** [comment]
As this is not a new issue, let's fix this in the 0.94k release.

**Comment by komeifukuda** [comment]
I noticed another similar (little) problem.  My COPYRIGHT is old.
17 #define dd_COPYRIGHT   "Copyright (C) 1996, Komei Fukuda, fukuda@ifor.math.ethz.ch"
18 #define dd_DDVERSION   "Version 0.94g (March 23, 2012)"
 
The line 17 should be
#define dd_COPYRIGHT   "Copyright (C) 2018, Komei Fukuda, fukuda@math.ethz.ch"
although my old email address still works.

Sorry for not noticing this earlier.
